packages/react/src/Footer/FooterMenuLink.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { AnchorHTMLAttributes, ForwardedRef, PropsWithChildren } from 'react'
+import { Icon } from '../Icon'
+
+export type FooterMenuLinkProps = {
+  icon?: Function
+} & PropsWithChildren<AnchorHTMLAttributes<HTMLAnchorElement>>
+
+export const FooterMenuLink = forwardRef(
+  ({ children, className, icon, ...restProps }: FooterMenuLinkProps, ref: ForwardedRef<HTMLAnchorElement>) => (
+    <li>
```

Comment:
Maybe we should add a `className` here, so users can easily style it if they want to. Similar to what we do in `Breadcrumb`

Reply:
We use the same approach in our [Table of Contents component](https://github.com/Amsterdam/design-system/blob/740104693186797e6e0fd79e3ad14fe47eaef972/packages/react/src/TableOfContents/TableOfContentsLink.tsx#L17). Let's add it, it makes the component more flexible for our users.

Reply:
Ah static class, I thought you meant moving the `className` prop here